RevenueCatUI/CustomerCenter/Data/CustomerCenterManagementOption.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterManagementOption.swift
+//
+//  Created by Cesar de la Vega on 11/3/25.
+
+import Foundation
+
+/// Protocol for action types that can be handled by the Customer Center
+public protocol CustomerCenterActionable {}
+
+/// A wrapper for CustomerCenterActionable that conforms to Equatable
+/// This allows us to use CustomerCenterActionable with SwiftUI's preference system
+public struct CustomerCenterManagementOptionWrapper: Equatable {
```

Comment:
I don't think `CustomerCenterManagementOptionWrapper` should be public
```suggestion
struct CustomerCenterManagementOptionWrapper: Equatable {
```